Lib/configparser.py

```diff
@@ -687,7 +688,7 @@ def read(self, filenames, encoding=None):
 
         Return list of successfully read files.
         """
-        if isinstance(filenames, str):
+        if isinstance(filenames, (str, bytes, os.PathLike)):
```

Comment:
I may be wrong but my understanding is that bytes are a valid type for paths. The PEP for Pathlib provides operators to convert to bytes - https://docs.python.org/3/library/pathlib.html#operators